licenses-binary/LICENSE-MIT-codehaus.txt

```diff
@@ -0,0 +1,21 @@
+The MIT License
```

Comment:
for which jar?

Reply:
animal-sniffer-annotations-1.21.jar
https://github.com/apache/incubator-kyuubi/blob/a54daf27c2798d99cc3fb9af7e4ac17b54d6a3e8/dev/dependencyList#L19